### PR TITLE
log selector w/ labels vs awk/xargs gymnastics

### PR DESCRIPTION
**Pull request by mchmarny**

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

File: BUILD.md [hunk 1]
```diff
@@ -118,9 +118,9 @@ to deploy it as part of KEDA. Do the following:
    ```
 4. Once the keda pods are up, check the logs to verify everything running ok, eg:
     ```bash
-    kubectl get pods --no-headers -n keda | awk '{print $1}' | grep keda-operator | xargs kubectl -n keda logs -f
-
-    kubectl get pods --no-headers -n keda | awk '{print $1}' | grep keda-metrics-apiserver | xargs kubectl -n keda logs -f
+    kubectl logs -l app=keda-operator -n keda -f
+
```
Comment (tomkerkhove) [inline BUILD.md:122]: This is failing static analysis
```suggestion

```

**Comment by tomkerkhove** [comment]
Can you fix DCO issus please? https://github.com/kedacore/keda/blob/v2/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

File: BUILD.md (hunk 1 above)
Comment (mchmarny) [reply BUILD.md]: Ye, the location of the whitespace is not clear, updated, hope this passes

**Review by zroubalik — APPROVED** [review]
LGTM, just the DCO

**Comment by mchmarny** [comment]
Clone, rebase, and push back to origin is a lot of work for 2 line doc change i made entirely in GH UI ;)

**Comment by tomkerkhove** [comment]
Thanks and sorry for the trouble!

**Pull request merged**